pytest_logger/plugin.py

```diff
@@ -164,13 +187,20 @@ def _refresh_link(source, link_name):
         pass
 
 
-def _make_logsdir(tmpdirhandler, logdirlinks):
+def _make_logsdir_tmpdir(tmpdirhandler):
     logsdir = tmpdirhandler.getbasetemp()
     if logsdir.basename.startswith('popen-gw'):
         logsdir = logsdir.join('..')
     logsdir = logsdir.join('logs').ensure(dir=1)
-    for link in logdirlinks:
-        _refresh_link(str(logsdir), link)
+    return logsdir
+
+
+def _make_logsdir_dir(dstname, cleandir=True):
```

Comment:
By 'dead code' I mean: you cannot cover `else` by tests as every pass through the function would have cleandir set to True. I suppose keeping old test logs could be achieved otherwise by using parametric tmpdir like this one:
https://github.com/aurzenligl/study/blob/master/python-tmpdir/test_x.py
This way one could point to a directory where tmpdirs would be generated in exactly the same fashion as in `/tmp/pytest-of-x` directory. This is how this could look if user gave `/tmp/foo` as the root logs directory:
```
/tmp/foo $ tree -L 1
.
├── logs-109
├── logs-110
├── logs-111
├── logs-112
└── logs-aurzenligl -> /tmp/foo/logs-112
```